FlipperZero/Shared/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,148 +1,260 @@
+      "size" : "40x40",
+      "idiom" : "watch",
+      "filename" : "AppIcon40x40@2x.png",
+      "subtype" : "38mm",
+      "role" : "appLauncher",
+      "scale" : "2x"
+    },
+    {
+      "size" : "86x86",
+      "idiom" : "watch",
+      "filename" : "AppIcon86x86@2x.png",
+      "subtype" : "38mm",
+      "role" : "quickLook",
+      "scale" : "2x"
+    },
+    {
+      "size" : "98x98",
+      "idiom" : "watch",
+      "filename" : "AppIcon98x98@2x.png",
+      "subtype" : "42mm",
+      "role" : "quickLook",
+      "scale" : "2x"
+    },
+    {
+      "size" : "50x50",
+      "idiom" : "watch",
+      "filename" : "AppIcon50x50@2x.png",
+      "subtype" : "44mm",
+      "role" : "appLauncher",
+      "scale" : "2x"
+    },
+    {
+      "size" : "44x44",
+      "idiom" : "watch",
+      "filename" : "AppIcon44x44@2x.png",
+      "subtype" : "40mm",
+      "role" : "appLauncher",
+      "scale" : "2x"
+    },
+    {
+      "size" : "108x108",
+      "idiom" : "watch",
+      "filename" : "AppIcon108x108@2x.png",
+      "subtype" : "44mm",
+      "role" : "quickLook",
+      "scale" : "2x"
+    },
+    {
+      "size" : "1024x1024",
+      "idiom" : "watch-marketing",
+      "filename" : "watchicon.png",
+      "scale" : "1x"
     },
     {
+      "size" : "16x16",
       "idiom" : "mac",
-      "scale" : "1x",
-      "size" : "16x16"
+      "filename" : "icon_16x16.png",
+      "scale" : "1x"
     },
     {
+      "size" : "16x16",
       "idiom" : "mac",
-      "scale" : "2x",
-      "size" : "16x16"
+      "filename" : "icon_16x16@2x.png",
+      "scale" : "2x"
     },
     {
+      "size" : "32x32",
       "idiom" : "mac",
-      "scale" : "1x",
-      "size" : "32x32"
+      "filename" : "icon_32x32.png",
+      "scale" : "1x"
     },
     {
+      "size" : "32x32",
       "idiom" : "mac",
-      "scale" : "2x",
-      "size" : "32x32"
+      "filename" : "icon_32x32@2x.png",
+      "scale" : "2x"
     },
     {
+      "size" : "128x128",
       "idiom" : "mac",
-      "scale" : "1x",
-      "size" : "128x128"
+      "filename" : "icon_128x128.png",
+      "scale" : "1x"
     },
     {
+      "size" : "128x128",
       "idiom" : "mac",
-      "scale" : "2x",
-      "size" : "128x128"
+      "filename" : "icon_128x128@2x.png",
+      "scale" : "2x"
     },
     {
+      "size" : "256x256",
       "idiom" : "mac",
-      "scale" : "1x",
-      "size" : "256x256"
+      "filename" : "icon_256x256.png",
+      "scale" : "1x"
     },
     {
+      "size" : "256x256",
       "idiom" : "mac",
-      "scale" : "2x",
-      "size" : "256x256"
+      "filename" : "icon_256x256@2x.png",
+      "scale" : "2x"
     },
     {
+      "size" : "512x512",
       "idiom" : "mac",
-      "scale" : "1x",
-      "size" : "512x512"
+      "filename" : "icon_512x512.png",
+      "scale" : "1x"
     },
     {
+      "size" : "512x512",
       "idiom" : "mac",
-      "scale" : "2x",
-      "size" : "512x512"
+      "filename" : "icon_512x512@2x.png",
```

Comment:
Obviously `512x512@2x` is the same icon as `1024x1024`. Because all image files are in the same directory, can we reuse them across the asset definition and remove duplicates?

Reply:
Too much of manual labor for me to reduce assets directory for no more than couple hundred KB.